tools/Setup-Test-Environment.ps1

```diff
@@ -0,0 +1,26 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [string]$service,
+    [string]$keyVault = "AzureSDK-Dev-TestSecrets"
+    )
+
+$secrets = (az keyvault secret list --vault-name $keyVault --output json | ConvertFrom-Json);
+$secrets | Write-Host;
+
+$languagePrefix = "net-";
+$prefix = $languagePrefix + $service + "-";
+
+foreach ($secret in $secrets)
+{
+    $id = $secret.id;
+    $name = $id.Substring($id.LastIndexOf('/') + 1);
+
+    if ($name.StartsWith($prefix))
+    {
+        $value = (az keyvault secret show --id $id | ConvertFrom-Json).value;
+        $variableName = $name.Substring($languagePrefix.Length).Replace('-', '_');
```

Comment:
I assume you also want to uppercase the variable to match the env convention.

Reply:
Will do